--- libedit-20160903-3.1.org/src/editline/readline.h	2017-01-25 02:55:04.271871165 +0000
+++ libedit-20160903-3.1/src/editline/readline.h	2017-01-25 02:56:31.759777416 +0000
@@ -136,6 +136,7 @@
 extern int		(*rl_getc_function)(FILE *);
 extern VFunction	*rl_redisplay_function;
 extern VFunction	*rl_completion_display_matches_hook;
+extern VFunction	*rl_event_hook;
 extern VFunction	*rl_prep_term_function;
 extern VFunction	*rl_deprep_term_function;
 extern int readline_echoing_p;
